### PR TITLE
[Update 2.5] Add mention of powered by header and how to configure it

### PR DESCRIPTION
**Pull request by andrerom**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://github.com/ezsystems/ez-support-tools/pull/52
| Versions      | 2.5+

This also drop Password expiry step as it is covered in database schema steps above.

### CODE REVIEW

**Comment by andrerom** [comment]
@DominikaK There was mention by @mnocon of also finding a place to document this in general, any suggestions on where this would fit?

File: docs/updating/4_update_2.5.md [hunk 1]
```diff
@@ -104,14 +104,25 @@ bin/console cache:pool:clear cache.redis
     CREATE INDEX ezpage_pages_content_id_version_no ON ezpage_pages(content_id, version_no);
     ```
 
-## Updating to 2.5.6
+## Updating to 2.5.15
 
-### Password expiration
+### Powered By Header
```
Comment (DominikaK) [inline docs/updating/4_update_2.5.md:109]:
```suggestion
### Powered-By header
```

File: docs/updating/4_update_2.5.md [hunk 2]
```diff
@@ -104,14 +104,25 @@ bin/console cache:pool:clear cache.redis
...
 
-Run the following script to update the database:
+In order to be able to promote use of eZ Platform, `ezsystems/ez-support-tools` v1.0.9 adds capability to set powered by header.
```
Comment (DominikaK) [inline docs/updating/4_update_2.5.md:111]:
```suggestion
In order to promote use of eZ Platform, `ezsystems/ez-support-tools` v1.0.9 sets the Powered-By header.
```
Or maybe, as we shouldn't expect users know which package has which version in which release:
```suggestion
In order to promote use of eZ Platform, `ezsystems/ez-support-tools` as of eZ Platform v2.5.15 sets the Powered-By header.
```

File: docs/updating/4_update_2.5.md [hunk 3]
```diff
@@ -104,14 +104,25 @@ bin/console cache:pool:clear cache.redis
...
+By default this is enabed, and will generate a header like `Powereed-By: eZ Platform Enterprise 2.5`.
```
Comment (DominikaK) [inline docs/updating/4_update_2.5.md:112]:
```suggestion
It is enabled by default and generates a header like `Powered-By: eZ Platform Enterprise 2.5`.
```

File: docs/updating/4_update_2.5.md [hunk 4]
```diff
@@ -104,14 +104,25 @@ bin/console cache:pool:clear cache.redis
...
 
-`mysql -u <username> -p <password> <database_name> < vendor/ezsystems/ezpublish-kernel/data/update/mysql/dbupdate-7.5.4-to-7.5.5.sql`
-
-or for PostgreSQL:
+If you'd like to ommit version number, for instance if you are stuck on a fast track release beyond it's maintenance window:
```
Comment (DominikaK) [inline docs/updating/4_update_2.5.md:114]:
```suggestion
To omit the version number, for instance if you are stuck on a fast track release beyond its maintenance window,
use the following configuration:
```

File: docs/updating/4_update_2.5.md [hunk 5]
```diff
@@ -104,14 +104,25 @@ bin/console cache:pool:clear cache.redis
...
+``` yaml
+ezplatform_support_tools:
+    system_info:
+        powered_by:
+            release: "none"
+```
 
-`psql <database_name> < vendor/ezsystems/ezpublish-kernel/data/update/postgres/dbupdate-7.5.4-to-7.5.5.sql`
+If you however want to ommit the whole feature, you can likewise do that by disabling it:
```
Comment (DominikaK) [inline docs/updating/4_update_2.5.md:122]:
```suggestion
To opt out of the whole feature, by disabling it with the following configuration:

```

**Review by DominikaK — CHANGES_REQUESTED** [review]

**Comment by DominikaK** [comment]
The best place we have for documenting the settings outside the upgrade section would be https://doc.ibexa.co/en/master/guide/config_repository/

**Review by DominikaK — APPROVED** [review]

**Comment by andrerom** [comment]
@DominikaK Updated to reflect change in code PR where default is now only showing major version, also update to target 2.5.16 as 2.5.15 is out already.

**Pull request merged**